lib/cretonne/src/verifier/mod.rs

```diff
@@ -409,6 +411,33 @@ impl<'a> Verifier<'a> {
                             got);
             }
         }
+        // We also verify if the postorder defined by `DominatorTree` is sane
+        for (index, (&true_ebb, &test_ebb)) in
+            self.domtree
+                .cfg_postorder()
+                .iter()
+                .zip(domtree.cfg_postorder().iter())
+                .enumerate() {
+            if true_ebb != test_ebb {
```

Comment:
I can use `==` but the drawback is that is doesn't return any interesting location when the two slices are different.